### ISSUE TITLE
:book: Add expand-all for hidden codeblocks

### ISSUE THREAD

**Issue by quick-question** [kubernetes-sigs/kubebuilder#972]
/kind bug

**Expected behavior:**
All pieces of necessary code in the kubebuilder book are obvious when following the tutorial.

**Actual behavior:**
Many necessary blocks of code are collapsed in an easy-to-miss way, even appearing to be comments.

**Reproduction steps:**
Go to: https://book.kubebuilder.io/cronjob-tutorial/controller-implementation.html
Look at the expandable segments labelled "// Clock" and "// ignoreNotFound"
(there are many other similar instances)

Is there any rationale for having this code hidden by default? It can be very frustrating when you are following the tutorial and have to backtrack because you missed code that was collapsed.

**Comment by DirectXMan12** [comment]
The idea is that they're not *super* important to the overall flow of the tutorial (they're mostly implementation details) -- you can mostly figure out what they do by name alone, and go read the details later.  Otherwise, they interrupted the overall "this is how you build a controller" flow a bit, I thought.

Did you have more detailed thoughts on the matter?  I'm definitely open to changing things (either specific cases or the general case).  `ignoreNotFound` needs to be changed to `client.IgnoreNotFound` anyway.

**Comment by quick-question** [comment]
I can see where you're coming from. I guess it depends on how you expect users to interact with the book. If the idea is that they should clone the sample project and dissect it by following along with the book, then it hiding those code blocks is intuitive. But if the intention is that they should follow along by replicating the sample project piece-by-piece, then I feel that having the code "hidden" just throws in another gotcha to trip up users.

**Comment by DirectXMan12** [comment]
I think that kinda depends on where a person is coming from.  The motivation for the tutorial was kinda two common questions:

1. "What're the steps from start to end to build a controller?  I've got my own logic to fill in, I just need the 'skeleton' step-by-step"
2. "How does that translate to a real controller?  I'll followed the skeleton, but it doesn't do anything!"

We used to just have question 1 covered, and got a lot of questions along the lines of question two, so the goal was kinda supposed to be "show a real example so that someone using this to build something concrete could follow along as a kind of getting started guide", while also showing real-world considerations, how to make use of the client, etc.

Perhaps the right words would be "reference by example".

In all honesty though (and this is gonna sound a bit weird), I find myself (pleasantly) surprised at the number of people actually following along without looking at the canonical repo.

Would an "expand all" button work or something?  I kinda want to preserve the "here's the important parts that you'll need to look at to build your own thing" aspect as the default, but hopefully "expand all" might make things easier if you're word-for-word following along?

/kind documentation
